patches/minecraft/net/minecraft/core/Registry.java.patch

```diff
@@ -140,15 +140,19 @@
        return StructureFeature.f_67014_;
     });
     public static final ResourceKey<Registry<StructurePieceType>> f_122842_ = m_122978_("worldgen/structure_piece");
-@@ -311,7 +_,7 @@
+@@ -311,10 +_,10 @@
     public static final ResourceKey<Registry<Codec<? extends SurfaceRules.RuleSource>>> f_194572_ = m_122978_("worldgen/material_rule");
     public static final ResourceKey<Registry<StructureProcessorType<?>>> f_122854_ = m_122978_("worldgen/structure_processor");
     public static final ResourceKey<Registry<StructurePoolElementType<?>>> f_122855_ = m_122978_("worldgen/structure_pool_element");
 -   public static final Registry<BlockStateProviderType<?>> f_122856_ = m_122999_(f_122846_, () -> {
 +   @Deprecated public static final Registry<BlockStateProviderType<?>> f_122856_ = forge(f_122846_, () -> {
        return BlockStateProviderType.f_68752_;
     });
-    public static final Registry<FoliagePlacerType<?>> f_122858_ = m_122999_(f_122848_, () -> {
+-   public static final Registry<FoliagePlacerType<?>> f_122858_ = m_122999_(f_122848_, () -> {
++   public static final Registry<FoliagePlacerType<?>> f_122858_ = forge(f_122848_, () -> {
```

Comment:
Please add a `@Deprecated` annotation to this modified field, as is done with the other modified registry fields.
```suggestion
+   @Deprecated public static final Registry<FoliagePlacerType<?>> f_122858_ = forge(f_122848_, () -> {
```

Reply:
Done